bokehjs/src/lib/models/widgets/tables/data_table.ts

```diff
@@ -34,6 +42,14 @@ export class TableDataProvider implements DataProvider<Item> {
 
   constructor(source: ColumnDataSource, view: CDSView) {
     this.init(source, view)
+    return new Proxy(this, {
```

Comment:
Can't use `Proxy` due to IE compat.

Reply:
I left a bunch of comments/questions for you that all seem to have disappeared :( Will readd them now.